08LessonEight/in-class-project/in-class.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title></title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+  <header>
+    <h1>
+      Recent From Blog
+    </h1>
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+    </p>
+  </header>
+  <main class="clearfix">
+
+
+    <article class = "bigbox left">
+      <div class="big_img">
+        <img src="img/keyboard.jpg">
```

Comment:
looks like you have a broken image here. You may have renamed the image "keyboard2.jpg" and still sourced "keyboard.jpg". Fix the broken link and I can bump you up to 100.